.eslintrc.js

```diff
@@ -1,8 +1,14 @@
 const { merge } = require( 'lodash' );
 const reactVersion = require( './client/package.json' ).dependencies.react;
+const path = require( 'path' );
 
 module.exports = {
 	root: true,
+	parserOptions: {
+		babelOptions: {
+			configFile: path.join( __dirname, './babel.config.js' ),
```

Comment:
Do we need to explicitly specify the path to Babel config? The new plugin no longer finds it in the default location?

Reply:
Without an explicit path it will search for `babel.config.js` in the current working directory. This will work for us most of the time, as we only run eslint related commands from the root.

However, using it as an absolute path guarantees that it will always work, even if someone invokes eslint from a different directory (eg: a subpackage). It also has the benefit of making an explicit mention to `babel.config.js`, so it is easier to found where that config is being used.

More info in https://babeljs.io/docs/en/config-files#project-wide-configuration and https://babeljs.io/docs/en/config-files#root-babelconfigjson-file